trillian/migrillian/main.go

```diff
@@ -117,6 +126,35 @@ func main() {
 	core.RunMigration(cctx, ctrls)
 }
 
+// NewClientDialOptionsFromFlags returns a list of grpc.DialOption values to be
+// passed as DialOption arguments to grpc.Dial. It configures TLS credentials
+// if a CA certificate file is specified, otherwise it uses insecure credentials.
+func NewClientDialOptionsFromFlags(backend string) ([]grpc.DialOption, error) {
+	dialOpts := []grpc.DialOption{}
+
+	if *tlsCACertFile == "" {
```

Comment:
Please flip these conditions so that the insecure version is done as the default else block. This makes it cleaner to implement any other TLS flags (e.g. support for self-signed certs that did not involve a CA). Specifically:

```go
if len(*tlsCACertFile) > 0 {
  ... set up CA cert pool and return creds
} else {
  ... return insecure version
}
```

Which allows this to naturally grow, e.g.:
```go
if len(*tlsCACertFile) > 0 {
  ... set up CA cert pool and return creds
} else if len(*tlsDirectCertFile) > 0 {
  ... return a version using credentials.NewClientTLSFromFile
} else {
  ... return insecure version
}
```